public/app/percona/settings/components/Platform/Connected/Connected.messages.ts

```diff
@@ -3,5 +3,7 @@ export const Messages = {
   connected: 'This PMM instance is connected to Percona Platform.',
   disconnect: 'Disconnect',
   modalTitle: 'Disconnect PMM from Percona Platform',
-  modalBody: 'Are you sure you want to disconnect from Percona Platform? This will automatically log you out from PMM.',
+  forceModalTitle: 'Force-disconnect PMM from Percona Platform',
```

Comment:
```suggestion
  forceModalTitle: 'Force disconnect PMM from Percona Platform',
```

Reply:
I am not sure about removing the dash. This is a text Catalina sent me and she has dash everywhere.